pkg/loghelper/loghelper.go

```diff
@@ -63,3 +66,36 @@ func SetLogFormat() {
 	}
 	logrus.SetFormatter(customFormatter)
 }
+
+func SetHooks() {
+	logrus.AddHook(&writer.Hook{ // Send logs with level higher than warning to stderr
+		Writer: os.Stderr,
+		LogLevels: []logrus.Level{
+			logrus.PanicLevel,
+			logrus.FatalLevel,
+			logrus.ErrorLevel,
+			logrus.WarnLevel,
+		},
+	})
+	logrus.AddHook(&writer.Hook{ // Send info and debug logs to stdout
+		Writer: os.Stdout,
+		LogLevels: []logrus.Level{
+			logrus.InfoLevel,
+			logrus.DebugLevel,
+		},
+	})
+}
+
+// setLogLevel sets the log level for logrus based on the "TNF_LOG_LEVEL" environment variable
+func SetLogLevel() {
+	params := configuration.GetTestParameters()
+
+	var logLevel, err = logrus.ParseLevel(params.LogLevel)
+	if err != nil {
+		logrus.Error("TNF_LOG_LEVEL environment set with an invalid value, defaulting to DEBUG \n Valid values are:  trace, debug, info, warn, error, fatal, panic")
```

Comment:
I thinks there's no documentation currently for TNF_LOG_LEVEL variable, may we add it to the docs?
Also, if saying that "defaulting to DEBUG", it looks like "DEBUG" is a correct value but it is "debug", should it be lower case instead?

Reply:
Adjusted to be lowercase `debug` and added a note in the docs.